aiagents/panel_utils/panel_utils.py

```diff
@@ -80,15 +81,15 @@ def on_chain_start(
     def on_chain_end(self, outputs: dict[str, Any], *args, **kwargs):
         print(dumps(outputs, indent=2))
         print(self.agent_name)
-        if "This output contains the appropriate swagger metadata file to use for the task at hand" in outputs["output"]:
+        if "this output contains the appropriate swagger metadata file to use for the task at hand" in outputs["output"].lower():
```

Comment:
can you explain the purpose of this?

Reply:
This is needed for multiple swagger API support and is where we figure out which swagger metadata file was selected for the task, that file name is then used as a key to find the corresponding API endpoint and bearer token.